### ISSUE TITLE
feature git-lfs failed to install in Devcontainer build

### ISSUE THREAD

**Issue by udaya2899** [devcontainers/features#888]
Today, we discovered that our devcontainer build is failing with the following error:
```
=> ERROR [dev_containers_target_stage  6/15] RUN --mount=type=bind,from=  1.8s
------
 > [dev_containers_target_stage  6/15] RUN --mount=type=bind,from=dev_containers_feature_content_source,source=git-lfs_2,target=/tmp/build-features-src/git-lfs_2     cp -ar /tmp/build-features-src/git-lfs_2 /tmp/dev-container-features  && chmod -R 0755 /tmp/dev-container-features/git-lfs_2  && cd /tmp/dev-container-features/git-lfs_2  && chmod +x ./devcontainer-features-install.sh  && ./devcontainer-features-install.sh  && rm -rf /tmp/dev-container-features/git-lfs_2:
#0 0.428 ===========================================================================
#0 0.428 Feature       : Git Large File Support (LFS)
#0 0.428 Description   : Installs Git Large File Support (Git LFS) along with needed dependencies. Useful for base Dockerfiles that often are missing required install dependencies like git and curl.
#0 0.428 Id            : ghcr.io/devcontainers/features/git-lfs
#0 0.428 Version       : 1.1.1
#0 0.428 Documentation : https://github.com/devcontainers/features/tree/main/src/git-lfs
#0 0.428 Options       :
#0 0.428     VERSION="latest"
#0 0.428     AUTOPULL="true"
#0 0.428 ===========================================================================
#0 0.472 Installing Git LFS...
#0 0.476 (*) No apt package for bookworm amd64. Installing manually.
#0 1.113 GIT_LFS_VERSION=3.5.0
#0 1.113 Looking for release artfact: git-lfs-linux-amd64-v3.5.0.tar.gz
#0 1.297 (*) Downloading GPG key...
#0 1.330 gpg: keybox '/tmp/tmp-gnupg/pubring.kbx' created
#0 1.517 gpg: /tmp/tmp-gnupg/trustdb.gpg: trustdb created
#0 1.520 gpg: key F1BA225C0223B187: public key "brian m. carlson <sandals@crustytoothpaste.net>" imported
#0 1.529 gpg: Total number processed: 1
#0 1.529 gpg:               imported: 1
#0 1.604 gpg: key F54FE648088335A9: public key "Chris Darroch (CODE SIGNING KEY) <chrisd@apache.org>" imported
#0 1.606 gpg: Total number processed: 1
#0 1.606 gpg:               imported: 1
#0 1.675 gpg: key ABA67BE5A5795889: public key "Lars Schneider <larsxschneider@gmail.com>" imported
#0 1.676 gpg: Total number processed: 1
#0 1.676 gpg:               imported: 1
#0 1.682 gpg: no valid OpenPGP data found.
#0 1.682 gpg: decrypt_message failed: Unknown system error
#0 1.683 ERROR: Feature "Git Large File Support (LFS)" (ghcr.io/devcontainers/features/git-lfs) failed to install! Look at the documentation at https://github.com/devcontainers/features/tree/main/src/git-lfs for help troubleshooting this error.
------
ERROR: failed to solve: executor failed running [/bin/sh -c cp -ar /tmp/build-features-src/git-lfs_2 /tmp/dev-container-features  && chmod -R 0755 /tmp/dev-container-features/git-lfs_2  && cd /tmp/dev-container-features/git-lfs_2  && chmod +x ./devcontainer-features-install.sh  && ./devcontainer-features-install.sh  && rm -rf /tmp/dev-container-features/git-lfs_2]: exit code: 2
```

We tried in a different machine and failing with the same error there too. I can confirm this was working 3 days ago. We use git-lfs in our devcontainer.json like this:
```
"ghcr.io/devcontainers/features/git-lfs:1": {
      "version": "latest"
    }
 ```

**Comment by cr2007** [comment]
Same issue here 🙋‍♂️
My Dev Container build failed on GitHub Codespaces.

The key error message appearing was this:
```
#16 5.004 gpg: no valid OpenPGP data found.
#16 5.004 gpg: decrypt_message failed: Unknown system error
```

Followed by saying that Git LFS feature failed to install

**Comment by samruddhikhandale** [comment]
Hi 👋 

Thanks for reporting, looks like we are running into similar issues as https://github.com/devcontainers/features/issues/334 ; where the `latest` tag available is `3.5.0`, however, the corresponding releases are still not available. 

You'd be able to temporary fix it by pinning to earlier version, see 👇 

```jsonc
"ghcr.io/devcontainers/features/git-lfs:1": {
      	"version": "3.4"
}
```

I'd shortly put up a PR to fix the root problem and add retry logic.

**Comment by Ahmed30mara2018** [comment]
> Today, we discovered that our devcontainer build is failing with the following error:
> ```
> => ERROR [dev_containers_target_stage  6/15] RUN --mount=type=bind,from=  1.8s
> ------
>  > [dev_containers_target_stage  6/15] RUN --mount=type=bind,from=dev_containers_feature_content_source,source=git-lfs_2,target=/tmp/build-features-src/git-lfs_2     cp -ar /tmp/build-features-src/git-lfs_2 /tmp/dev-container-features  && chmod -R 0755 /tmp/dev-container-features/git-lfs_2  && cd /tmp/dev-container-features/git-lfs_2  && chmod +x ./devcontainer-features-install.sh  && ./devcontainer-features-install.sh  && rm -rf /tmp/dev-container-features/git-lfs_2:
> #0 0.428 ===========================================================================
> #0 0.428 Feature       : Git Large File Support (LFS)
> #0 0.428 Description   : Installs Git Large File Support (Git LFS) along with needed dependencies. Useful for base Dockerfiles that often are missing required install dependencies like git and curl.
> #0 0.428 Id            : ghcr.io/devcontainers/features/git-lfs
> #0 0.428 Version       : 1.1.1
> #0 0.428 Documentation : https://github.com/devcontainers/features/tree/main/src/git-lfs
> #0 0.428 Options       :
> #0 0.428     VERSION="latest"
> #0 0.428     AUTOPULL="true"
> #0 0.428 ===========================================================================
> #0 0.472 Installing Git LFS...
> #0 0.476 (*) No apt package for bookworm amd64. Installing manually.
> #0 1.113 GIT_LFS_VERSION=3.5.0
> #0 1.113 Looking for release artfact: git-lfs-linux-amd64-v3.5.0.tar.gz
> #0 1.297 (*) Downloading GPG key...
> #0 1.330 gpg: keybox '/tmp/tmp-gnupg/pubring.kbx' created
> #0 1.517 gpg: /tmp/tmp-gnupg/trustdb.gpg: trustdb created
> #0 1.520 gpg: key F1BA225C0223B187: public key "brian m. carlson <sandals@crustytoothpaste.net>" imported
> #0 1.529 gpg: Total number processed: 1
> #0 1.529 gpg:               imported: 1
> #0 1.604 gpg: key F54FE648088335A9: public key "Chris Darroch (CODE SIGNING KEY) <chrisd@apache.org>" imported
> #0 1.606 gpg: Total number processed: 1
> #0 1.606 gpg:               imported: 1
> #0 1.675 gpg: key ABA67BE5A5795889: public key "Lars Schneider <larsxschneider@gmail.com>" imported
> #0 1.676 gpg: Total number processed: 1
> #0 1.676 gpg:               imported: 1
> #0 1.682 gpg: no valid OpenPGP data found.
> #0 1.682 gpg: decrypt_message failed: Unknown system error
> #0 1.683 ERROR: Feature "Git Large File Support (LFS)" (ghcr.io/devcontainers/features/git-lfs) failed to install! Look at the documentation at https://github.com/devcontainers/features/tree/main/src/git-lfs for help troubleshooting this error.
> ------
> ERROR: failed to solve: executor failed running [/bin/sh -c cp -ar /tmp/build-features-src/git-lfs_2 /tmp/dev-container-features  && chmod -R 0755 /tmp/dev-container-features/git-lfs_2  && cd /tmp/dev-container-features/git-lfs_2  && chmod +x ./devcontainer-features-install.sh  && ./devcontainer-features-install.sh  && rm -rf /tmp/dev-container-features/git-lfs_2]: exit code: 2
> ```
> 
> We tried in a different machine and failing with the same error there too. I can confirm this was working 3 days ago. We use git-lfs in our devcontainer.json like this:
> ```
> "ghcr.io/devcontainers/features/git-lfs:1": {
>       "version": "latest"
>     }
>  ```
>
